components/infobox/wikis/freefire/infobox_team_custom.lua

```diff
@@ -0,0 +1,77 @@
+---
+-- @Liquipedia
+-- wiki=freefire
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'staff' then
+		return {
+			Cell{name = 'Founders',	content = {_team.args.founders}},
+			Cell{name = 'CEO', content = {_team.args.ceo}},
+			Cell{name = 'Manager', content = {_team.args.manager}},
+			Cell{name = 'Team Captain', content = {_team.args.captain}},
+			Cell{name = 'Coaches', content = {_team.args.coaches}},
+			Cell{name = 'Analysts', content = {_team.args.analysts}},
+		}
+	end
+	return widgets
+end
+
+function CustomTeam:createBottomContent()
+	return Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Placement summary',
+		{team = _team.name or _team.pagename}
+	) .. Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing tournaments of',
+		{team = _team.name or _team.pagename}
+	)
+end
+
+function CustomTeam:addToLpdb(lpdbData, args)
+	if not String.isEmpty(args.teamcardimage) then
+		lpdbData.logo = args.teamcardimage
+	elseif not String.isEmpty(args.image) then
+		lpdbData.logo = args.image
+	end
+
+	lpdbData.region = Variables.varDefault('region', '')
```

Comment:
why the empty string as fallback?
imo nil would be better
```suggestion
	lpdbData.region = Variables.varDefault('region')
```

Reply:
Having nil values can cause weird things in some situation, with the result being the previous value would be kept

Reply:
was it empty string before?
does lpdb even set empty string at all from wiki code?

Reply:
[Infobox team](https://liquipedia.net/freefire/TemplateArchive:Infobox_team)
yeah, it was empty before

Reply:
> was it empty string before? does lpdb even set empty string at all from wiki code?

tbh, not sure